Matlab_Scripts/5_Depth_Colourless/Study/README.md

```diff
@@ -0,0 +1,12 @@
+Server locations
+
+Data: M:\Data\VR_P3_Five_Depth_Colourless
+
+Analysis: M:\Analysis\VR_P3_Five_Depth_Colourless
+
+Experiment: M:\Experiments\VR_Five_Depth_Colorless\Experiment
+
+Refer to markerinfo.txt for trigger info
+
+Refer to subNotes for non-identifying info on each participant (do not add identifiers!)
```

Comment:
current there is ID ing information in there, please remove the names of the participants